mautic/mautic
# Added Mandrill webhook support for their spam and unsubscribed events

**Description**

This PR adds support for Mandrill's spam and unsubscribed webhook events.

Fixes #1158.

**Testing**
Testing requires a public accessible installation of Mautic.

1. Configure Mautic to send through Mandrill. 
2. Create a lead with the email example.webhook@mandrillapp.com.  
3. Setup Mandrill webhook https://mautic.org/docs/emails/bounce_management.html.  For testing, setup one event per webhook.  Click mandrill's "send test" button for the first event.  Easiest way is to check the email_donotemail table for an entry for that lead.  Delete the entry in the table then try the next event.
